data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,25 +1417,59 @@ mission "Hai Reveal [A14] A Time of Trouble"
+					goto "begin the meeting"
+				`	"Can the Hai really pass through, with the Republic Navy blocking their way?"`
+					goto "navy blocks hai"
+				`	"What if we let them through, to attack the pirate planets?"`
+					goto "let hai through"
+				`	"What if we solve the kidnapping problem instead of the Hai doing it?"`
+					goto "hai concerns"
+				`	"Who in the Cabinet is blocking progress?"`
+					goto "who is blocking"
+			label "navy blocks hai"
+			`	Xilin responds, "The Navy has blockaded the wormhole to the best of its ability, but the reality is that the Navy is already heavily committed, and barely look to be a match for the Hai fleet numbers. The Hai, on the other hand, do not yet appear to be committed heavily." He looks at Alondo briefly. "We had the unstated privilege of observing the Hai security fleet left behind on Everhope in addition to what we saw beyond the wormhole on our way in."`
 			`	Danforth interjects, "In short, we doubt we can stop them while we're stretched out here... and if the reality is that they're only going to go after pirate worlds... well, we'd be under orders to disengage. The political crisis would be more tolerable than the probable military defeat."`
-			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy. It would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood The Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
-			`	"They haven't outright said it... but they've implied that the Navy could end up killing the Hai involved in the process by not understanding the capabilities of the Hai, or by placing too great an emphasis on preserving Navy lives. Enough people in the Cabinet disagree, that we're sort of at an impasse, for the moment."`
-			`	"Who in the Cabinet is blocking it?" The speaker is Turner, and everyone turns to look at him with a mixture of irritation or surprise.`
+			`	Xilin nods grimly. "However, that would be a drastic failure of foreign policy."`
+				goto questions
+			label "let hai through"
+			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood The Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
```

Comment:
```suggestion
			`	Xilin replies, "That would undermine the legitimacy of the Republic, especially after the war we've just had, and it would be impossible to realistically assure people that an invasion is really not going to happen. True or not. In all likelihood the Republic would fracture and the Free Worlds would probably be on the receiving end of a migration crisis. Parliament has tried to say that they'll take care of it themselves, but the Hai say they've had enough of sitting around and want to be there to save their own people.`
```